test/integration/targets/inventory/1/2/3/extra_vars_relative.yml

```
@@ -0,0 +1,11 @@
+- hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Make sure var is loaded
+      assert:
+        that:
+          - my is defined
+          - my == 'var'
+
+    - name: Reload inventory, forces extra vars re-eval from diff basedir
+      meta: refresh_inventory
```

We should make sure the reload did succeed with some additional asserts:

```suggestion
    - name: Make sure inventory was loaded properly
      assert: &inventory_load_assert_condition
        that:
          - "'webservers' in groups"
          - "'web_host.example.com' in groups['webservers']"

    - name: Reload inventory, forces extra vars re-eval from diff basedir
      meta: refresh_inventory
      
    - name: Make sure inventory was reloaded properly
      assert: *inventory_load_assert_condition
```

In order to make this work, we will also need some additional inventory sources. A simple `base_inventory.ini` containing the `web_host.example.com` line should work just fine.

---

not sure why you need 2nd inventory, it will behave the same as with the first

---

If there is no "base" inventory, the constructed plugin has nothing to chew and will not produce a webservers group. This basic inventory provides some data for the plugin and allows us to determine if the constructed plugin did its thing successfully.